Rebuild with libpqxx 6

beenje opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Reconnection mechanism is gone in libpqxx 7.x.
It should be implemented in the hdbpp-timescale library.

See https://gitlab.com/tango-controls/hdbpp/libhdbpp-timescale/-/issues/33#note_2727264158

I had to remove compilation with cpptango 10.x as there are conflicts with OpenTelemetry.
Will have to re-enable it when we can move back to version 7.x

beenje committed:
Rebuild with libpqxx 6

Reconnection mechanism is gone in libpqxx 7.x.
It should be implemented in the hdbpp-timescale library.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

beenje committed:
Remove cpptango 10.x to rebuild with libpqxx 6

There are conflicts between opentelemetry and that version of libpqxx.
Will have to move to libpqxx 7 to compile with tango 10.

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.1 and conda-forge-pinning 2025.09.04.02.19.31

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6
